app/permissions/team.rb

```diff
@@ -0,0 +1,119 @@
+Canaid::Permissions.register_for(Team) do
+  # view projects, view protocols
+  # leave team, view team users (ATWHO)
+  # view samples, export samples
+  # view repositories, view repository, export repository rows
+  can :read_team do |user, team|
+    user.is_member_of_team?(team)
+  end
+
+  # edit team name, edit team description
+  can :update_team do |user, team|
+    user.is_admin_of_team?(team)
+  end
+
+  # invite user to team, change user's role, remove user from team
+  can :manage_user_team do |user, team|
```

Comment:
How about `can_manage_user_teams?`, or even `can_manage_team_users?`. I like the latter, it's not "correct" per-se if we base names from actual `ActiveRecord` names, but it tells you exactly what the permission is about.